fieldservice/models/fsm_location.py

```diff
@@ -64,6 +64,22 @@ def _tz_get(self):
                                      compute='_compute_equipment_ids')
     sublocation_count = fields.Integer(string='Sub Locations',
                                        compute='_compute_sublocation_ids')
+    location_heirarchy = fields.Char(string='Location Heirarchy',
+                                compute='_compute_location_heirarchy')
+
+    def compute_loc(self, parents):
+        if self.fsm_parent_id:
+            parents = str(self.fsm_parent_id.name) + '/' + parents
+            parents = self.fsm_parent_id.compute_loc(parents)
+        return parents
+
+    def _compute_location_heirarchy(self):
+        parents = ""
+        for loc in self:
+            parents = loc.compute_loc(parents)
+            parents = parents + str(loc.name)
+            loc.location_heirarchy = parents
+            parents = ""
```

Comment:
I think you try to do something like this:
https://github.com/odoo/odoo/blob/11.0/addons/product/models/product.py#L36

Reply:
@max3903 : This change has been push to current PR only.